rust/blockstore/src/provider.rs

```diff
@@ -100,6 +100,15 @@ impl BlockfileProvider {
         };
         Ok(())
     }
+
+    pub async fn prefetch(&self, id: &uuid::Uuid) -> Result<usize, Box<dyn ChromaError>> {
```

Comment:
this is a new top-level API, not attached to a reader or writer, so we can avoid K & V generics